src/functionalTest/java/org/jfrog/gradle/plugin/artifactory/TestConstant.java

```diff
@@ -61,6 +63,9 @@ public class TestConstant {
     // Version catalog
     public static final String MIN_GRADLE_VERSION_CATALOG_VERSION = "7.0";
 
```

Comment:
I noticed this class is named `TestConstant`. I recommend renaming it to `TestConsts` (plural form instead of singular).